### DISCUSSION TITLE
Did someone know about SPEA2 ? the algo give different answer when i input same mo files

### DISCUSSION THREAD

**Discussion by lijunliangTG** [OpenModelica/OpenModelica#9682] [Q&A]
the mo file is as follows

```mo
model functionTest
  constant Real pi=3.1415926;
  parameter Real x(min=-1,max=2) = 0;
  Real y;
equation
  y=x*sin(10*pi*x)+2.0;
end functionTest;
```

Could you please help me find the same optimization result every time I enter the same MO file

**Comment by casella** [comment]
@lijunliangTG I don't get the point of your question. The model you wrote has a constant solution `x = 0`, `y = 2`. What optimization are you referring to?

**Reply by lijunliangTG** [reply]
SPEA2
